cmd/create/execution.go

```diff
@@ -0,0 +1,123 @@
+package create
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	"github.com/lyft/flytectl/cmd/config"
+	cmdCore "github.com/lyft/flytectl/cmd/core"
+	"github.com/lyft/flyteidl/gen/pb-go/flyteidl/admin"
+	"github.com/lyft/flyteidl/gen/pb-go/flyteidl/core"
+
+	"github.com/google/uuid"
+	"github.com/pkg/errors"
+)
+
+const (
+	executionShort = "Create execution resources"
+	executionLong  = `
+Create the executions for given launchplan in a project and domain.
+Currently the launchplan and where the execution needs to be launched , need to belong to same domain and project.
+
+There are three steps in generating an execution.
+
+- Generate the execution param yaml file containing in the default input parameters for the launch plan.
+- If the default parameters exist then you can modify the default param values in the generated yaml file.
+- Run the execution passing in the generated yaml file.
+
+The following command would generate core.basic.lp.go_greet.inputs.yaml file by using the launch plan provided in argument.
+This would use the latest version of the launch plan.
+::
+
+ bin/flytectl create execution core.basic.lp.go_greet  -d development  -p flytesnacks -g
```

Comment:
it should also support passing in a task. This is called as a singletask execution.

Reply:
flytekit i dont see it supporting launching tasks. Let me know if i am missing something here 
https://github.com/flyteorg/flytekit/blob/c02075d472b5587d199630bcfc7f9937673c6a0e/flytekit/clis/flyte_cli/main.py#L1044
The resourceType returned as part of the previous line
lp_id = _identifier.Identifier.from_python_std(urn)
is ignored when passing it to fetch launchPlan.

The UI is doing it by passing in the roles
eg : by launching from the UI i had to pass this role
arn:aws:iam::173840052742:role/flytefunctionaltestsbatchworker-production-iad
This option is not available in flytekit.
Do we want to accept this as part of the yaml file  for flytectl  or as explicit argument when launching an execution for a task .

Reply:
yes, for now we will have to accept a role as an input. If CLI is not doing that then its a miss. Dont rely on it.
So I mentioned earlier that we want to use the inputs.yaml file to have more than inputs - it should be all the meta inputs like role - all these fields and probably more in the future - https://github.com/flyteorg/flyteidl/blob/master/protos/flyteidl/admin/execution.proto#L216-L244

Reply:
Done. its now support tasks and workflow executions both.